actionlist/index.android.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * Sample React Native App
```

Comment:
Your application is no longer the Sample React Native App, so this comment should be removed